include/OpenSpaceToolkit/Astrodynamics/EventCondition/DurationCondition.hpp

```diff
@@ -0,0 +1,74 @@
+/// Apache License 2.0
+
+#ifndef __OpenSpaceToolkit_Astrodynamics_EventConditions_DurationCondition__
+#define __OpenSpaceToolkit_Astrodynamics_EventConditions_DurationCondition__
+
+#include <OpenSpaceToolkit/Core/Types/Real.hpp>
+#include <OpenSpaceToolkit/Core/Types/String.hpp>
+
+#include <OpenSpaceToolkit/Mathematics/Objects/Vector.hpp>
+
+#include <OpenSpaceToolkit/Physics/Time/Duration.hpp>
+
+#include <OpenSpaceToolkit/Astrodynamics/EventCondition.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace eventcondition
```

Comment:
Man I totally thought we had an example of 2 word namespaces in OSTk but I guess not. Couldn't find one, so I'll refer to the google style guide.
https://google.github.io/styleguide/cppguide.html#Namespace_Names

Will leave it up to you if you want to follow this, but they recommend `event_condition` over `eventcondition`